src/ui/components/TwoButtonGroup/TwoButtonGroup.stories.tsx

```diff
@@ -33,6 +34,6 @@ Default.args = {
     selected: "one",
     oneLabel: "Find",
     twoLabel: "Create",
-    oneIcon: true,
-    twoIcon: true,
+    oneIcon: true as never,
```

Comment:
why is never being used here? never is for code that should be unreachable

Reply:
to silence TS warning

Reply:
arent there better types that you can use like unknown?

Reply:
there is an error on `unknown` type

Reply:
In your opinion, what is better to set?

Reply:
Id need to test but i dont really have time, if you dont wanna see that pass